src/i18n/strings/en_EN.json

```diff
@@ -699,6 +699,7 @@
     "%(senderName)s: %(reaction)s": "%(senderName)s: %(reaction)s",
     "%(senderName)s: %(stickerName)s": "%(senderName)s: %(stickerName)s",
     "Change notification settings": "Change notification settings",
+    "LaTeX math in messages": "LaTeX math in messages",
```

Comment:
We use UK English for the untranslated string, which I believe should be "LaTeX maths ..."  here and in the other translation files, and in `settings/Settings.ts`  (The en_US translation would be "LaTeX math ...")